systemtests/tests/volume-pruning-test/testrunner

```diff
@@ -0,0 +1,71 @@
+#!/bin/sh
+#
+# Run a simple backup
+#   then restore it.
+#
```

Comment:
I don't really understand what this test does, and this comment doesn't describe it.

Reply:
Good point